src/aux_utils.cpp

```diff
@@ -808,12 +839,17 @@ namespace diskann {
           ROUND_UP(npts_64, n_data_nodes_per_sector) / n_data_nodes_per_sector;
     }
     _u64 disk_index_file_size =
-        (n_sectors + n_reorder_sectors + 1) * SECTOR_LEN;
+        (n_sectors + n_reorder_sectors + 1) * SECTOR_LEN;   
 
     // write first sector with metadata
     *(_u64 *) (sector_buf.get() + 0 * sizeof(_u64)) = disk_index_file_size;
     *(_u64 *) (sector_buf.get() + 1 * sizeof(_u64)) = npts_64;
     *(_u64 *) (sector_buf.get() + 2 * sizeof(_u64)) = medoid;
+    /*if(reorder_data){
+      *(_u64 *) (sector_buf.get() + 2 * sizeof(_u64)) = medoid;  
```

Comment:
excellent point, we need to enter the reordered medoid here.. why is this commented?

Reply:
Commented this code earlier to verify. Now I have added the reordered medoid.